examples/crowd-funding/src/contract.rs

```diff
@@ -44,54 +44,84 @@ impl Contract for CrowdFunding<ViewStorageContext> {
 
     async fn execute_operation(
         &mut self,
-        _context: &OperationContext,
+        context: &OperationContext,
         operation_bytes: &[u8],
     ) -> Result<ExecutionResult, Self::Error> {
         let operation: Operation =
             bcs::from_bytes(operation_bytes).map_err(Error::InvalidOperation)?;
 
+        let mut result = ExecutionResult::default();
+
         match operation {
             Operation::PledgeWithTransfer { owner, amount } => {
```

Comment:
Should this just be called `Pledge` now?

Reply:
Yes possibly

Reply:
but it would be not as consistent with the `Effect` though